css/quiz.css

```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css?family=Rokkitt');
+
+
+body {
+	width: 60%;
+	margin: 0 auto;
```

Comment:
It's not a good idea to size the body. Setting global fonts, font sizes, background color, etc are fine, but if you want to create  a container of a certain dimension, it's better to wrap your content in a wrapper div. Just give it the class of `content` or `wrapper` or something and then use these sizing properties on it.